### ISSUE TITLE
ACF missing LSB for user profile

### ISSUE THREAD

**Issue by manh-trinhquoc** [qtranslate/qtranslate-xt#1270]
I create custom field at user profile page (wp-admin/profile.php) and it not working at all. The languague switching button is not showing.

![image](https://user-images.githubusercontent.com/34735335/209497281-366bc965-92a3-40b5-b37c-05712604d0ca.png)
![image](https://user-images.githubusercontent.com/34735335/209497291-ff266c3d-7052-4d8b-8ae5-a0159caee57e.png)

I use wordpress 6.6.1 and qtranslate-xt 3.12.1

**Comment by herrvigg** [comment]
Up to qTranslate-XT 3.12.1 you were supposed to add that `profile.php` page in ACF options, but I don't know if this used to work. There were many bugs due to incomplete configuration.

There are many fixes for ACF with qTranslate-XT 3.13.0 and the page list have been removed from the options. Unfortunately the profile.php page is missing... but the fix is very easy now, I pushed a fix to `master` and this works.

So you have two options:
1. Download latest [master.zip](https://github.com/qtranslate/qtranslate-xt/archive/refs/heads/master.zip)
or
2. Install QT-XT 3.13.0 (download or github updater) and fix it locally by adding `profile.php => ''` in this list
https://github.com/qtranslate/qtranslate-xt/blob/e79ecf00c4c7053078496f200e57495896e153bf/modules/acf/src/qtx_module_acf_admin.php#L102-L110
